[레벨 2 글쓰기 미션] 찰리(박민영) 미션 제출합니다.

Gomding opened the pull request:
안녕하세요 찰리입니다!

부족한 점이 많은 글이라 가차없는 피드백 부탁드립니다!!!

[편하게 보기](https://github.com/Gomding/woowa-writing-3/blob/gomding/LEVEL2.md)

Gomding committed:
docs(LEVEL1.md) : Level 1 글쓰기

Gomding committed:
글쓰기 미션 1단계 작성

Gomding committed:
LEVEL1.md 수정

Gomding committed:
gather 의 장점을 리스트화

Gomding committed:
Update LEVEL1.md

Gomding committed:
Update LEVEL1.md

Gomding committed:
미션이 있는것은 -> 미션이 있다는 것은 수정

Gomding committed:
과거의 약속들이 어떻게 에너지를 소모시켰는지 구체화

Gomding committed:
Code With Me 의 단점에 대한 예시 추가

Gomding committed:
오타수정

Gomding committed:
게더의 장점을 추가

Gomding committed:
모호한 표현 수정

Gomding committed:
Code With Me 에서 겪었던 문제를 비유를 통해 설명

Gomding committed:
Update LEVEL1.md

Gomding committed:
Update LEVEL1.md

Gomding committed:
지쳐버렸습니다 소주제 마지막 부분의 어색한 부분 수정

Gomding committed:
Update LEVEL1.md

Gomding committed:
전사팀 -> 전사교육팀 으로 수정

Gomding committed:
글쓰기 시작

Gomding committed:
글쓰기 레벨2 작성 완료 (맞춤법 /문법검사까지)

365kim commented on LEVEL2.md:
```diff
@@ -0,0 +1,83 @@
+구글에 검색해도 안 나와서 시간이 많이 허비될 수 있고,<br>
+블로그의 글이 틀린 정보일 수 있다는 것입니다. (이것은 잘못된 정보라는 바이러스를 가지고 다른 사람한테 전파하는 효과가 생길 수 있습니다)<br>
+또한 자신의 블로그에 학습한 걸 글로 정리함으로 완전히 내 것으로 만들었다고 생각했는데 이것은 완전히 틀린 생각이었습니다.<br>
+블로그 글은 단순히 저장소에 저장했을 뿐 내용에 대해서는 이해 했을수도 이해 못했을 수도 있었습니다.<br>
```
`이해 했을수도` => `이해했을 수도` 로 변경되면 좋을 것 같습니다 :)

국립국어원 답변([-하다 띄어쓰기](https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=146792), [-수도 띄어쓰기](https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=146792)) 인용해봅니다!
```
동사 '하다'를 써서 '공부를 하다'와 같이 띄어 쓸 수 있고, 
'공부를 하다'에서 조사 '를'을 쓰지 않는다면 접사 '-하다'가 쓰인 것으로 보아 붙여 적습니다. 
보기를 들어, '공부하다, 공부를 하다', '청소하다, 청소를 하다'와 같이 씁니다.
```
```
'일 수도 있다, 할 수도 있다'에서 '-ㄹ'은 어미이며 '수'는 의존 명사, '도'는 보조사입니다. 
따라서 '일 수도, 할 수도'로 띄어 쓰는 것이 바르며 이는 한 단어가 아닙니다.
```

dusdn1702 commented on LEVEL2.md:
```diff
@@ -0,0 +1,83 @@
+
+우테코에서 생긴 학습법의 변화는 아래와 같습니다.<br>
+* 모르면 스스로 알 때까지 찾아봐 -> 모르면 찾아보고 오랜 시간을 허비하면 물어보자<br>
+* 학습한 걸 글로 정리하면 내 것이야 -> 학습한 걸 다른 사람한테 설명할 줄 알아야 내 것이야<br>
```
오오 멋집니다 ㅎㅎ

dusdn1702 commented on LEVEL2.md:
```diff
@@ -0,0 +1,83 @@
+* 모르면 스스로 알 때까지 찾아봐 -> 모르면 찾아보고 오랜 시간을 허비하면 물어보자<br>
+* 학습한 걸 글로 정리하면 내 것이야 -> 학습한 걸 다른 사람한테 설명할 줄 알아야 내 것이야<br>
+
+이제부터 과거에 혼자 공부하던 때의 학습법과 우테코에서 변화한 학습법을 비교하려고 합니다.<br>
```
이미 앞에서 `->`를 통한 변화를 준지라 문장이 어색하게 느껴지는 것 같아요!
이제부터 이 변화들에 대해 알아보려고 합니다.
이제부터 이 변화를 비교하려고 합니다.
와 같이 바꿔보는 건 어떤가요??

사실 제 말도 어색하게 느껴지니 찰리가 읽어보고 괜찮다면 무시하셔도 좋습니다 ㅎㅎ

Gomding replied:
읽어보니 바꿔보는게 자연스러워서 이제부터 이 변화를 비교하려고 합니다. 로 바꿔봤어요! 😄

365kim commented on LEVEL2.md:
```diff
@@ -0,0 +1,83 @@
+
+> 단순히 질문하고 듣는 것은 구글에 검색한 뒤 하나의 블로그 글을 읽는 것과 큰 차이가 없습니다.
+<br>
+## 학습 한걸 글로 정리하면 내 것이야 -> 학습한 걸 다른 사람한테 설명할 줄 알아야 내 것이야
```
![image](https://user-images.githubusercontent.com/60066472/120064674-636c1b00-c0a8-11eb-89b0-8dca56821108.png)

여기 md파일 문법 적용이 안되서 깨져보이네요...!

Gomding replied:
😮  ## 이 그대로 보였군요 ㅠㅠ 바로 수정했습니다!!

365kim commented on LEVEL2.md:
```diff
@@ -0,0 +1,83 @@
+
+위의 학습법은 몇 가지 문제가 있습니다.<br>
+구글에 검색해도 안 나와서 시간이 많이 허비될 수 있고,<br>
+블로그의 글이 틀린 정보일 수 있다는 것입니다. (이것은 잘못된 정보라는 바이러스를 가지고 다른 사람한테 전파하는 효과가 생길 수 있습니다)<br>
```
`바이러스`라는 은유가 정말 찰떡이네요...! 👍 제가 혹시 잘못된 정보를 전파하지는 않았을까 되돌아보게 되는 문장이었어요 :)

dusdn1702 commented on LEVEL2.md:
````diff
@@ -0,0 +1,83 @@
+<b>모르면 스스로 알 때까지 계속 찾아봐</b>라는 공부법은 이런 배경에 의해 탄생했습니다.<br>
+<br>
+우테코에 와서도 한동안은 누군가에게 질문하는 것이 어색해서 이런 공부법은 계속 이어졌습니다.<br>
+하지만 함께 고민해보자는 크루 또는 페어 프로그래밍을 진행하면서 질문에 답변하다 보니<br>
+점점 생각에 변화가 생겼습니다, ```누군가에게 질문하고 의견을 나누면 다른 사람의 생각도 흡수 할 수 있는걸?``` 이라고 생각하게 됐습니다.<br>
````
```suggestion
하지만 함께 고민해보자는 크루 또는 페어 프로그래밍을 진행하면서 질문에 답변하다 보니 점점 생각에 변화가 생겼습니다. <br>
```누군가에게 질문하고 의견을 나누면 다른 사람의 생각도 흡수 할 수 있는걸?``` 이라고 생각하게 됐습니다.<br>
```
위와 같이 변경하는건 어떨까요?? 갑자기 문장이 줄바꿈으로 끊겨서 흐름이 끊기는 느낌이 들었어요!

Gomding replied:
갑자기 끊어져버리니 너무 어색했네요 😮 
샐리의 의견대로 한 줄이 되도록 수정했어요! 👍

dusdn1702 commented on LEVEL2.md:
```diff
@@ -0,0 +1,83 @@
+* 아.. 정리했는데 기억이 안 나네요
+
+굉장히 부끄러운 상황입니다.<br>
+부끄러운 건 둘째치고 학습한 걸 설명하지 못한다는 건 너무 심각했습니다.<br>
```
왜 심각한지도 덧붙이면 좋을 것 같아요!
예를 들어 학습이 제대로 이루어지지 않았다는 것이니~ 와 같이?

Gomding replied:
심각한 이유를 추가해봤습니다 👍 

```
이건 지식을 이해하려고 한 게 아니라 단순히 글을 따라 적었다는걸 의미했습니다.
```

ddongule replied:
같이 부끄러워하고 갑니다 하하핫!!!

365kim commented on LEVEL2.md:
````diff
@@ -0,0 +1,83 @@
+
+혼자 공부할 때 이 말을 조금 뒤틀린 방식으로 적용하고 있었습니다.<br>
+
+```모르겠으면 물어본다 == 나약하다``` 라고 생각했습니다.<br>
````
![image](https://user-images.githubusercontent.com/60066472/120064760-e5f4da80-c0a8-11eb-9830-de7b19fe15ba.png)
여기부터 파란색으로 표시된 곳 부터 깨지고(?) 있는 듯 합니다...!

365kim commented on LEVEL2.md:
```diff
@@ -0,0 +1,83 @@
+이전에는 블로그에 한편의 글을 쓰면 이제 해당 내용에 대한 모든 것을 설명할 수 있으리라 생각했습니다.<br>
+하지만 실제로 해당 내용에 대한 크루의 질문이 들어오면 두 가지 중 하나의 반응을 보였습니다.<br>
+
+* 그런 글을 내가 썼어요?
+* 아.. 정리했는데 기억이 안 나네요
```
혹시... 저세요...? 🥲

Gomding replied:
공감 감사합니다.. 😭

ddongule replied:
여기 그루밍이 또있었네요 찰루밍이신가요?

dusdn1702 reviewed (approved):
멋져요 찰리!
학습법을 멋있게 찾아가시고 있는 것 같네요 ㅎㅎㅎ
몇가지 피드백 남겨두었는데 별게 아닌지라 바로 approve 드릴게요~!

레벨2까지 수고하셨습니다🙌 레벨3도 화이팅💪

365kim commented on LEVEL2.md:
```diff
@@ -0,0 +1,83 @@
+아마 평생 학습법에 대한 정답을 찾을 수 없을지도 모르고 없을 수도 있습니다.<br>
+확실한 건 자신에게 맞는 학습법이 가장 정답에 근접할 수 있다고 생각합니다.<br>
+<br>
+<b>현재 저의 목표는 상대방의 질문에 최대한 친절하고 이해하기 쉽게 설명하는 것</b>입니다.<br>
+어려운 개념을 쉽게 설명한다는 것은 그만큼 개념을 잘 이해하고 있다는 것을 증명하기도 합니다.<br>
+잘 이해한 것이 아니라면 쉽게 설명하려다가 계속 설명이 중단되는 현상을 자주 경험했습니다.<br>
+<br>
+이를 보완하기 위해 앞으로도 학습법을 찾기 위한 여정은 계속될 것입니다!
```
친절한 찰리~! 👍 독자로 하여금 앞으로의 성장이 기대되게끔 하는 마지막 문장이네요! ㅎㅎ

365kim commented on LEVEL2.md:
```diff
@@ -0,0 +1,83 @@
+> 조금 기적의 계산법 같지만 다른 크루가 10시간 동안 고민하고 정리했던 걸 들으면 10시간을 이득 본셈입니다.<br>
+
+물론 고민이 생기자마자 물어보는 것은 독이 됩니다.<br>
+고민에 대해 내가 찾아보고 생각해보고 (애매한 내용이라도)정리한 뒤에 질문하는 크루에게 내 의견이 어떤지 정도는 말할 수 있어야<br>
```
![image](https://user-images.githubusercontent.com/60066472/120064939-d3c76c00-c0a9-11eb-9250-5e0d1caa4052.png)

혹시 문장이 끝나지 않았는데 개행을 추가해주신 이유가 있을까용,,,??? 일부러 호흡을 넣어주신 의도라고 생각해도 조금 어색한 것 같아서 코멘트 남겨봅니다...!

365kim commented on LEVEL2.md:
```diff
@@ -0,0 +1,83 @@
+# 우테코에서 찾은 나만의 효율적인 공부법
+<br>
+우테코에 오기 전 학습법은 정말 단순합니다.
```
이 문장은 `단순했습니다.`와 같이 종결어미를 과거형으로 써도 괜찮을 것 같아용!

365kim commented on LEVEL2.md:
````diff
@@ -0,0 +1,83 @@
+혼자 공부할 때 이 말을 조금 뒤틀린 방식으로 적용하고 있었습니다.<br>
+
+```모르겠으면 물어본다 == 나약하다``` 라고 생각했습니다.<br>
+<b>모르면 스스로 알 때까지 계속 찾아봐</b>라는 공부법은 이런 배경에 의해 탄생했습니다.<br>
````
저는 글을 다 읽고 나니 이 부분이 가장 기억에 남더라구요! 일화를 들려주신게 재밌기도 했고, 찰리가 이전에 왜 그런 생각을 갖게 되었는지 엿볼 수 있어서 좋았습니다 :) 👍

365kim reviewed (approved):
찰리, 안녕하세요 🙋🏻‍♀️
`리자돌림` 찰리의 좋은 글 잘 읽었습니당! ㅎㅎ

테크코스 초반에 `보이는 라디오` 미션에서 만난 이후로는 거의 4개월 만이네요!
찰리의 글에 그동안 본인에게 맞는 효과적인 학습법을 찾기 위해 어떤 고민을 거쳐 어떻게 성장해왔는지가 잘 드러나서 좋았습니다. 과거의 학습법과 현재의 학습법을 이해하기 쉽게 대조해주셔서 글의 흐름을 따라가기 쉬웠어요! 👍 

코멘트를 남기긴 했지만 글의 흐름을 크게 바꾸거나 할 만한 것은 아니어서 Approve 하겠습니다!
고생하셨습니다 😆

Gomding committed:
스타일 적용 안된곳 수정

sihyung92 commented:
찰리 하위~ 지나웨
지금보다 더 친절해질 수 있습니까???
모자바 스터디 때 부터 인연이 지금까지 이어지고 있는 중인데
찰리는 마주하는 사람으로 하여금 세상 편안하게 만들어주는 성격과 (인간 시몬스)
꾸준히 노력하는 부분이 정말 뛰어난 사람이라고 생각해여
좋은 지식을 전파하는 친절한 찰리 기대해 보겠습니다아
남은 우테코 기간도 화팅~~!

qhals321 commented on LEVEL2.md:
```diff
@@ -0,0 +1,89 @@
+군대에 입대하고 훈련소에서 조교분에게 들었던 말 중 가장 기억에 남는 말이 있습니다.<br>
+<br>
+
+> #### 뭘 찾아오라고 시키면 끝까지 찾아보고 그래도 모르겠으면 물어봐
```
으으으 악몽이 떠오른다..!!

Gomding replied:
😱 😱 😱

qhals321 commented on LEVEL2.md:
````diff
@@ -0,0 +1,89 @@
+<br>
+우테코에 와서도 한동안은 누군가에게 질문하는 것이 어색해서 이런 공부법은 계속 이어졌습니다.<br>
+하지만 함께 고민해보자는 크루 또는 페어 프로그래밍을 진행하면서 질문에 답변하다 보니<br>
+점점 생각에 변화가 생겼습니다, ```누군가에게 질문하고 의견을 나누면 다른 사람의 생각도 흡수 할 수 있는걸?``` 이라고 생각하게 됐습니다.<br>
````
정말 우테코는 함께 고민할 수 있다는 점이 좋은 것 같아요!!

Gomding replied:
함께 고민하고 공유하는게 너무 좋아요!!

qhals321 commented on LEVEL2.md:
```diff
@@ -0,0 +1,89 @@
+어려운 개념을 쉽게 설명한다는 것은 그만큼 개념을 잘 이해하고 있다는 것을 증명하기도 합니다.<br>
+잘 이해한 것이 아니라면 쉽게 설명하려다가 계속 설명이 중단되는 현상을 자주 경험했습니다.<br>
+<br>
+이를 보완하기 위해 앞으로도 학습법을 찾기 위한 여정은 계속될 것입니다!
```
여정... 멋지네요! 항상 응원해요 찰리!!

Gomding replied:
나봄도 항상 응원해요 !!!

ddongule replied:
여정에 함께해도 될까요?!! 
동료가 되어보시죠 
![dongryo](https://user-images.githubusercontent.com/26598561/120162073-fd5fcf00-c232-11eb-97ab-5265aa6bd4a0.jpeg)

Gomding replied:
![image](https://user-images.githubusercontent.com/57378410/120453027-a17d7d80-c3cd-11eb-8e66-8ccdf9f7040a.png)

qhals321 reviewed (approved):
글 너무나 잘 읽었어요!! 
스터디때도 항상 찰리 덕분에 같이 고민할 수 있었는데!!
찰리덕분에 저도 같이 고민하는 학습을 경험할 수 있었어요~~!!
글에 찰리가 너무 잘 표현되어있어서 찰리에 대해 더 알아간 기분이네요! 잘 읽었습니다~!!

Gomding committed:
오류 나는 부분 수정

Gomding committed:
이해 했을수도 -> 이해했을 수도 / 이해못했을 수도 -> 이해 못했을 수도 변경

Gomding committed:
우테코에 오기 전 학습법은 정말 단순합니다. -> 단순했습니다. 로 변경

Gomding committed:
이미 앞에서 언급한 내용은 간단하게 나타내도록 변환

Gomding committed:
이유가 모호한 문장에 이유를 추가

ddongule commented on LEVEL2.md:
```diff
@@ -0,0 +1,104 @@
+우테코에 오기 전 학습법은 정말 단순했습니다.
+<br>
+
+* 알고 싶은 게 있으면 책을 사서 읽는다.
+* 책을 읽다가 모르는 개념이 있으면 구글에 검색해본다.
+* 한국어로 정리된 블로그의 글을 읽는다.
+* 글을 참고해서 내 블로그에 글을 정리한다.
```
저도 다른 사람들의 글을 보고 마치 내 것인양 블로그에 글을 정리하곤 했던 것 같아요 하하핫!!!

ddongule commented on LEVEL2.md:
```diff
@@ -0,0 +1,104 @@
+
+이로 인해 자연스럽게 개념에 관한 공부를 열심히 하고 블로그 글보다는 영어로 된 공식 문서를 자주 보게 됐습니다.
+
+또한 블로그에 학습 내용이 저장되는 게 아닌 머리속에 학습 내용이 정리되어 저장됐습니다.
```
```
또한 블로그에 학습 내용이 저장되는 게 아닌 머리속에 학습 내용이 정리되어 저장됐습니다.
```
요 문장에서 약간 어색함을 느끼는데요!!!!
```
블로그에 단순히 학습 내용을 옮겨적는게 아니라 머리속에 정리된 내용들을 적을 수 있었습니다.
```
요런 뉘앙쓰로 한번 바꾸어보시는 건 어떨까요?! 
지금도 좋지만 바꾼다면 조금 더 확실한 의미전달이 될 수 있을 것 같아효 >__<

Gomding replied:
😮  좋은 피드백 감사합니다!
덕분에 훨씬 좋은 문장이 됐네요 👍 👍 👍

ddongule reviewed (approved):
안녕하세요 챨리!!!!!!!!!!!!!
그룸이입니다. 
제가 정말 학습한 내용을 글에 적는게 아니라 여기저기에 있는 편파적인 지식들을 끌어모아 하나의 글을 작성하고 있었던 제 자신에 대해 돌아보게되는 글이었어요.. 
반성하구 갑니다!!!!!
찰리의 글을 보고 앞으로 어떻게 글을 작성해나가야 할 지에 대해서도 고민할 수 있어서 좋았습니다!!!!
>> ㅑ!!!!!!!!!!!!!!!
코멘트를 하나 남기긴 했지만 큰 문제는 없는 것 같아서 Approve 남기구 갑니다 >__ <ㅎㅎㅎ

bperhaps commented on LEVEL2.md:
```diff
@@ -0,0 +1,104 @@
+위의 학습법은 몇 가지 문제가 있습니다.
+
+구글에 검색해도 안 나와서 시간이 많이 허비될 수 있고,<br>
+블로그의 글이 틀린 정보일 수 있다는 것입니다. (이것은 잘못된 정보라는 바이러스를 가지고 다른 사람한테 전파하는 효과가 생길 수 있습니다)<br>
```
이 문제는 정말 심각하다고 생각해요.... 일례로 어느 신기술에 대한 글을 구글에 치면 90퍼센트는 잘못된 내용이 나왔답니다... 요즘은 그나마 괜찮아 진 것 같았지만요...

bigsaigon333 replied:
역시 믿고 볼건 공식문서 밖에 없는걸까요 :dizzy_face:

Gomding replied:
공식문서 최고 영어 잘하고싶네요 😿

bperhaps commented on LEVEL2.md:
```diff
@@ -0,0 +1,104 @@
+현재는 우테코에 들어오고 학습법에 변화가 생겼습니다.<br>
+변화의 원인은 역시 환경의 변화라고 생각합니다.<br>
+모르는 내용이 있으면 혼자 고민하지 않고 함께 고민해줄 크루들이 있습니다.<br>
+또한 우테코에서 어떤 것이 정답인가요? 라고 물어보면 "정답을 찾기보단, 자신만의 정답을 찾고 자신이 찾은 정답을 다른 사람에게 설명할 줄 알아야 한다" 라는 말을 해줍니다.<br>
```
자신만의 losophy를 찾아가자구용~

bperhaps commented on LEVEL2.md:
```diff
@@ -0,0 +1,104 @@
+
+우테코에서 생긴 학습법의 변화는 아래와 같습니다.<br>
+* 모르면 스스로 알 때까지 찾아봐 -> 모르면 찾아보고 오랜 시간을 허비하면 물어보자
+* 학습한 걸 글로 정리하면 내 것이야 -> 학습한 걸 다른 사람한테 설명할 줄 알아야 내 것이야
```
정말 중요하다고 생각합니다!

bperhaps commented on LEVEL2.md:
````diff
@@ -0,0 +1,104 @@
+우테코에 와서도 한동안은 누군가에게 질문하는 것이 어색해서 이런 공부법은 계속 이어졌습니다.<br>
+하지만 함께 고민해보자는 크루 또는 페어 프로그래밍을 진행하면서 질문에 답변하다 보니<br>
+점점 생각에 변화가 생겼습니다, ```누군가에게 질문하고 의견을 나누면 다른 사람의 생각도 흡수 할 수 있는걸?``` 이라고 생각하게 됐습니다.<br>
+가장 큰 장점은 다른 크루가 고민하고 정리한 지식을 짧은 시간에 흡수 할 수 있다는 것입니다.<br>
````
다 뽑아 먹자구요!

Gomding replied:
손너잘 잘 뽑아 먹을게요 😉

bperhaps commented on LEVEL2.md:
```diff
@@ -0,0 +1,104 @@
+* 아.. 정리했는데 기억이 안 나네요
+
+굉장히 부끄러운 상황입니다.<br>
+부끄러운 건 둘째치고 학습한 걸 설명하지 못한다는 건 너무 심각했습니다.<br>
```
미안해 형....

Gomding replied:
아니야 좋았어 👍 👍 👍 👍 👍 👍

bperhaps reviewed (approved):
촬리~~ 나만의 작은 형....
학습로그 회의도 같이 하구 level1 면접도 같이 하면서 여러가지 의견을 나눴었지...
형이 어떤 고민을 하고 있는데, 그래서 어떻게 공부하고 있는지 옆에서 다 보여서 글이 더 공감가게 읽혔어..
항상 화이팅이야!!

bigsaigon333 commented on LEVEL2.md:
```diff
@@ -0,0 +1,104 @@
+# 우테코에서 찾은 나만의 효율적인 공부법 🧑‍🎓
+<br>
```
그냥 쓸데없는 코멘트 하나 남겨봅니다... (완전히 무시하셔도 됩니다)

--- 

 글 전체적으로 <br> 이 많은데, 이건 사용하시는 마크다운 에디터 때문에 생긴걸까요 아니면 개행을 강제하기 위해 직접 입력하신 걸까요?
 
 markdown linter 에 따르면 inline html 을 쓰는 건 권장되지 않는다고 합니당.. 아마 pure 하게 markdown 만 보는 사람에게는 불필요한 문구가 보여지기 때문인 것 같아요 
 https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md033---inline-html

Gomding replied:
😮  정말 좋은 정보 감사합니다

bigsaigon333 commented on LEVEL2.md:
```diff
@@ -0,0 +1,104 @@
+
+구글에 검색해도 안 나와서 시간이 많이 허비될 수 있고,<br>
+블로그의 글이 틀린 정보일 수 있다는 것입니다. (이것은 잘못된 정보라는 바이러스를 가지고 다른 사람한테 전파하는 효과가 생길 수 있습니다)<br>
+또한 자신의 블로그에 학습한 걸 글로 정리함으로 완전히 내 것으로 만들었다고 생각했는데 이것은 완전히 틀린 생각이었습니다.<br>
```
```suggestion
또한 자신의 블로그에 학습한 걸 글로 정리하여 완전히 내 것으로 만들었다고 생각했는데 이것은 완전히 틀린 생각이었습니다.<br>
```
는 어떨까요?!

Gomding replied:
매의눈 동동 감사합니다!

bigsaigon333 commented on LEVEL2.md:
````diff
@@ -0,0 +1,104 @@
+
+혼자 공부할 때 이 말을 조금 뒤틀린 방식으로 적용하고 있었습니다.<br>
+
+```모르겠으면 물어본다 == 나약하다``` 라고 생각했습니다.
````
```suggestion
```모르겠으면 물어본다 === 나약하다``` 라고 생각했습니다.
```

`===` 로 써주시면 더 좋겠네요 ㅎㅎ (농담)

365kim replied:
동동의 의견에 동의합니다! 🙋🏻‍♀️

Gomding replied:
이 부분 실제로 3개로 했다가 2개로 바꿨는데
역시 불편하셨군요 ㅠㅠ..

bigsaigon333 reviewed (approved):
커밋을 하나씩 차근차근 쌓으시면서 글을 작성하신 게 매우 인상적이었습니다.

저도 혼자서만 고민하는 편인데,  충분히 고민한 후에 다른 크루들과의 대화를 통해 남이 고민한 걸 흡수할 수 있다는 우테코의 장점을 제가 전혀 못살리고 있는 것 같아 아쉽다는 생각이 드네요...  

고생하셨습니다 👏🏼👏🏼👏🏼👏🏼

Gomding committed:
문장 수정

또한 블로그에 학습 내용이 저장되는 게 아닌 머리속에 학습 내용이 정리되어 저장됐습니다.

->

블로그에 단순히 학습 내용을 옮겨적는게 아니라 머리속에 정리된 내용들을 적을 수 있었습니다.

Gomding committed:
html 문법 제거하기

Gomding committed:
docs(LEVEL3.md) : Level 3 글쓰기
